kaldb/src/test/java/com/slack/kaldb/server/ZipkinServiceTest.java

```diff
@@ -152,6 +153,7 @@ public static List<LogWireMessage> generateLogWireMessagesForOneTrace(
   }
 
   @Test
+  @Ignore // Flakey test, occasionally returns an empty result
```

Comment:
Instead of ignoring it's best to run flaky tests under a flag and only run them locally or as a separate build step.

Reply:
As discussed, we'll circle back on this as part of a separate effort to fix the test suites. This isn't flaky as part of this PR and has been having issues for a few weeks since its original introduction (see - https://github.com/slackhq/kaldb/actions/runs/3316587545/jobs/5478551341 for an example from a few weeks ago on `master`).